lib/apple_sign_in/helpers/conditions/aud_condition.rb

```diff
@@ -7,8 +7,10 @@ def initialize(jwt)
         @aud = jwt['aud']
       end
 
-      def valid?
-        @aud == AppleSignIn.config.apple_client_id
+      def validate!
+        return true if @aud == AppleSignIn.config.apple_client_id
+
+        raise JWTValidationError, 'jwt_aud is different to apple_client_id'
```

Comment:
WDYT to use I18N here? just adding a config/locales folder, with en.yml file 😄

Reply:
what about doing this in a separate PR? just to don't make prs last forever

Reply:
That will add a dependency on the gem, I think this kind of error messages should be just for logging and the developer, not to show to the user so I don't think i18n would add value. To the user the developer should show a generic message, to not gave enough clues to possible bad actors.